### PR TITLE
Add option for choosing custom avatar icon in botbuilder

### LINKED ISSUE #553: Add option for choosing avatar in botbuilder
**Actual Behaviour**

Currently, user can either set a custom color or put a link to an avatar.

**Expected Behaviour**

The user should also be able to choose from a list of avatars (like SUSI, etc).

**Screenshots of the issue**

image

**Would you like to work on the issue?**
Yes.

### PR DESCRIPTION
**Pull request by hkedia321**
Fixes #553 

Changes: 
- add 10 custom avatars icons in `public/customAvatars` folder (downloaded from https://openclipart.org/).
- add option in the design section of `/botbuilder` to choose from these avatar icons.
- fix small bug in `susi-chatbot.js` to show the avatar icon in all bot icons in the messages.

Surge Deployment Link: [susi-chatbot-custom-avatars.surge.sh](http://susi-chatbot-custom-avatars.surge.sh/)

Screenshot for the change:
image
image

### CODE REVIEW

File: src/components/BotBuilder/BotBuilder.css [hunk 1]
```diff
@@ -60,6 +60,31 @@
...
+	margin-right: 10px;
+	cursor: pointer;
+}
+.icon-wrap{
+	display: inline-block;
+	position: relative;
+}
+.icon-wrap .tick{
+	display: none;
+    position: absolute;
```
Comment (PrP-11) [inline src/components/BotBuilder/BotBuilder.css]: Please see indentation here, its not aligned with lines above and below.

File: src/components/BotBuilder/BotBuilder.css [hunk 2]
```diff
@@ -60,6 +60,31 @@
...
+    top: -7px;
+    right: 3px;
+    font-size: 23px;
+    color: #4285f5;
+}
+.icon-selected .tick{
+	display: block;
+}
```
Comment (PrP-11) [inline src/components/BotBuilder/BotBuilder.css]: here

File: src/components/BotBuilder/BotBuilder.css [hunk 3]
```diff
@@ -60,6 +60,31 @@
...
+.icon-selected .bot-avatar{
+	border:solid 2px #4285f5;
```
Comment (PrP-11) [inline src/components/BotBuilder/BotBuilder.css]: and here. Thanks

**Comment by hkedia321** [comment]
@mariobehling I have updated the icons. All icons are now downloaded from https://openclipart.org/ and their source links are given in the `Utils/avatars.js` file too. Please review.

File: src/Utils/avatars.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+const host = 'https://skills.susi.ai';
```
Comment (mariobehling) [inline src/Utils/avatars.js]: Please don't hardcode the URL of the system.

Comment (hkedia321) [reply src/Utils/avatars.js]: @mariobehling made it dynamic using `window.location`. Please check.

**Pull request merged**